Makefile

```diff
@@ -13,12 +13,40 @@
 # limitations under the License.
 
 # Make will use bash instead of sh
+APP_NAME=forseti-security
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.4.1
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.4.6
 DOCKER_IMAGE_DEVELOPER_TOOLS := cft/developer-tools
 REGISTRY_URL := gcr.io/cloud-foundation-cicd
 
+# grep the version from the mix file
+VERSION=$(grep -o '[0-9].[0-9][0-9].[0-9]' google/cloud/forseti/__init__.py)
+
+# docker build --target forseti-server -t gcr.io/$PROJECT_ID/forseti-server:master .
+docker_server:
```

Comment:
unrelated to org/access policy changes.. I added this in while developing locally and think it's a good addition.

Reply:
Can we separate this change into a different PR, if this is something separate and not a dependency here?  The reasons are #1 it will make it easier to rollback if there are any problems (otherwise, the whole change here has to be rolled back) and #2 it will be easier for folks to find the change when troubleshooting (i.e. if we are scanning the PR titles to detect what has been changed recently).

Reply:
Reverted